### ISSUE TITLE
[fix] pfe-modal accessibility

### ISSUE THREAD

**Issue by starryeyez024** [patternfly/patternfly-elements#515]
1. The focus does not stay inside the modal 
2. The focus skips the close button

**Comment by castastrophe** [comment]
- [x] The aria-labelledby attribute on the dialog isn't receiving the ID of the heading properly (it's "undefined").
- [ ] The tabindex of the dialog/modal should be -1 instead of 0. This allows for programmatic focus only, and removes the "wrapper" from the normal tab order. PF4 does this much like the majority of the recommended patterns. (PR #1735)

> We should consider/discuss the possibility of allowing the tab focus to travel to the address bar as a future improvement. This hasn't been widely accepted yet (at least to my knowledge), but I've seen a few recommendations here, and I'd like to test this idea at some point.

Are there any actionable items we can address now or does this require a new issue and further discussion?

**Comment by bennypowers** [comment]
I _think_ this is solved in pfe 2.0 but would appreciate some more 👀  on it:

https://patternflyelements.org/components/modal/demo/

**Comment by bennypowers** [comment]
It appears this issue is no longer relevant. Please open a new issue if it is still relevant.
